### DISCUSSION TITLE
请求增加在当前标签页打开视频的功能

### DISCUSSION THREAD

**Discussion by AlanMaxxx** [the1812/Bilibili-Evolved#1367] [功能建议 / Ideas]
在B站搜索视频后每次点击搜索结果都会新增一个标签页打开视频，不一会儿就会导致打开很多无用的标签页，不仅需要手动关闭，而且也占用了内存资源，完全是浪费资源，所以能否增加一个直接在当前页面打开视频的功能，而不需要点击一个视频就打开一个标签页。

**Comment by the1812** [comment]
这个得浏览器控制了, 有的浏览器可以设置点击链接时是否打开新标签页

**Reply by AlanMaxxx** [reply]
> 这个得浏览器控制了, 有的浏览器可以设置点击链接时是否打开新标签页

可是有的网站浏览完之后再点击其中的一个连接就会打开新的标签页，其实完全没必要，只需要再原标签页打开就行！
不过最后我**解决**了！使用这个[这个脚本](https://greasyfork.org/en/scripts/4416-open-links-in-current-tab/discussions/71328)可以消除我的问题！
